examples/kms/vault/kms-config.yaml

```diff
@@ -9,6 +9,7 @@ data:
         "vaultAddress": "http://vault.default.svc.cluster.local:8200",
         "vaultAuthPath": "/v1/auth/kubernetes/login",
         "vaultRole": "csi-kubernetes",
+        "vaultToken": "sample_root_token_id",
```

Comment:
please add E2E testing for this one.

Reply:
Add this new configuration to value.yaml and kms-config.yaml

Reply:
@Madhu-1 this is `kms-config.yaml` , I failed to get which files you are referring ?

Reply:
ah correct, still one place its missing https://github.com/ceph/ceph-csi/blob/master/internal/util/vault.go#L57

Reply:
well, its source code comment. but I can add it there too..

Reply:
@Madhu-1 regarding e2e, I dont think it make sense to add some tests for wrong token or something like that. Its about vault configuration and we dont want to validate that.  btw, our current E2E example use this anyway.

Reply:
@Madhu-1 ` current E2E doesn't use vault user-provided token-based authentication, without E2E (how can we catch if we introduce a regression in this area?)` -> It does use vault token based authentication in this PR test as I mentioned in my previous comment! . That said, with the addition of vault token, the current E2E is using token based authentication. 

However while sending the PR, I thought of adding one more deployment model based on `disabling vault token` in the e2e. But, the deployment happens in a common path ( beforeEach) for our E2E, so ignored it. As  a proof that it has not regressed, I can remove the example config map entry and keep the code in place so that it validates regression too. Is that fine ?